tests/snmpsim/sophos4.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.1.0|4|Linux g4dc0e52 
```

Comment:
We could look at using `&& preg_match('/Linux ([\S]+) ([0-9\-.]{1,11})(g[\w\d]{7})(.*)/', $sysDescr)`

That matches the test data I found from an old issue: https://gist.github.com/laf/67cee069eb3db4af26dccdb67a7da3f5 but we'd need to find others potentially.